src/lib/snyk-to-html.ts

```diff
@@ -83,9 +85,35 @@ function metadataForVuln(vuln: any) {
     description: vuln.description || 'No description available.',
     fixedIn: vuln.fixedIn,
     packageManager: vuln.packageManager,
+    version: vuln.version,
+    cvssScore: vuln.cvssScore,
+    cveSpaced: cveSpaced || 'No CVE found.',
+    cveLineBreaks: cveLineBreaks || 'No CVE found.',
+    disclosureTime: dateFromDateTimeString(vuln.disclosureTime || ''),
+    publicationTime: dateFromDateTimeString(vuln.publicationTime || ''),
+    license: vuln.license || undefined
```

Comment:
Nit: you might not need to have a fallback to undefined? Unless you are worried about null?

Reply:
i think i've seen some erroneous nulls at times so im going to leave this for now